target/linux/ath79/image/common-mikrotik.mk

```diff
@@ -1,6 +1,6 @@
 define Device/mikrotik
 	DEVICE_VENDOR := MikroTik
-	DEVICE_PACKAGES := rbcfg
+	DEVICE_PACKAGES := 
```

Comment:
Remove this line entirely, it's initialized here: https://github.com/openwrt/openwrt/blob/master/include/image.mk#L376